asterisk/build-asterisk.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+PROGNAME=$(basename $0)
+
+if test -z ${ASTERISK_VERSION}; then
+    echo "${PROGNAME}: ASTERISK_VERSION required" >&2
+    exit 1
+fi
+
+set -ex
+
+mkdir /usr/src/asterisk
+cd /usr/src/asterisk
+
+curl -vsL http://downloads.asterisk.org/pub/telephony/asterisk/releases/asterisk-${ASTERISK_VERSION}.tar.gz |
+    tar --strip-components 1 -xz
+
+# 1.5 jobs per core works out okay
+: ${JOBS:=$(( $(nproc) + $(nproc) / 2 ))}
+
+./configure  --with-resample --with-jansson-bundled
+
+make menuselect/menuselect menuselect-tree menuselect.makeopts
+
+# disable BUILD_NATIVE to avoid platform issues
+menuselect/menuselect --disable BUILD_NATIVE menuselect.makeopts
+
+# enable good things
+menuselect/menuselect --enable BETTER_BACKTRACES menuselect.makeopts
+menuselect/menuselect --enable codec_opus menuselect.makeopts
+
+# download more sounds
+for i in CORE-SOUNDS-EN MOH-OPSOUND EXTRA-SOUNDS-EN; do
+    for j in ULAW ALAW G722 GSM SLN16; do
+        menuselect/menuselect --enable $i-$j menuselect.makeopts
+    done
+done
+
+make -j ${JOBS} all
+make install
+chown -R asterisk:asterisk /var/*/asterisk
+chmod -R 750 /var/spool/asterisk
+mkdir -p /etc/asterisk/
+
+# install example configuration
+make samples
+
+# set runuser and rungroup
+sed -i -E 's/^;(run)(user|group)/\1\2/' /etc/asterisk/asterisk.conf
+
+cd /
+
+# Uncomment this if you want to remove the asterisk source files.
+#exec rm -rf /usr/src/asterisk
```

Comment:
you can remove "exec" from this line

Reply:
removed the whole line as we are using multistage now.